compiler/src/iree/compiler/Codegen/LLVMCPU/LLVMCPUSplitReduction.cpp

```diff
@@ -67,6 +67,11 @@ LogicalResult splitReductionPrecondition(Operation *op,
     LLVM_DEBUG(llvm::dbgs() << "doesn't have exactly 1 input\n");
     return failure();
   }
+  if (linalgOp.hasIndexSemantics()) {
```

Comment:
Could we write how to fix this here, maybe? I think you had a comment on the GitHub issue. A link to that also would be good.

Reply:
Yeah, that would help. It's not clear to me what is missing